README.md

```diff
@@ -1 +1,154 @@
-# expense_tracker_RoR
\ No newline at end of file
+# Expense_tracker_RoR
+
+# 📗 Table of Contents
+
+- [expense_tracker_RoR](#expense_tracker_RoR)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Expense_tracker_RoR ](#-expense_tracker_RoR-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+    - [📝 License](#-license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blog_app <a name="about-project"></a>
+
+**Expense_tracker_RoR** Expense_tracker_RoR app is a website that allows users to manage personal finances. Featuring secure awuthentication with Devise. It enables users to create their own expenses for different categories and all financial data remains private and secure.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Programming languages</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+ </details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+    <li><a href="https://github.com/">Github</a></li>
+    <li><a href="https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow">Gitflow</a></li>
+  </ul>
+ </details>
+ <details>
+  <summary>Frameworks</summary>
+  <ul>
+    <li><a href="https://getbootstrap.com/">Bootstrap 5</a></li>
+    </ul>
+ </details>
+<details>
+  <summary>Modules</summary>
+ </details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href='https://heroku.com/'>Heroku</a></li>
+    </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Add User, Categoty and Expense
+- List all Categories and Expenses
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/haliljon/expense_tracker_RoR.git
+
+  cd expense_tracker_RoR
+
+  bundle install
+
+  rails c
+
+  rspec spec --format documentation
+```
+
+### 🚀 Live Demo
+
+- [Demo Video](https://drive.google.com/file/d/1N2PBY6U7CWSM0wSaKKQyAnjFB2bzKilW/view?usp=sharing)
```

Comment:
- [ ] As you also mentioned in your note the application is not yet deployed, from what I can tell form the screenshots I can only advise you to update your Ruby version. if that doesn't work please request for a 1 on 1 session with a senior CR, please have a look at [this article](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-Can-I-Get-Help-From-Code-Reviewers-) in case you would like to do so

Reply:
Hi @mikethreels, I checked the link for 1:1 sessions, all of them have time from Monday. It looks like I'm running out of time. Is there any way to extend the deadline for another day?